src/SamplesApp/SamplesApp.UnitTests.Shared/Controls/UITests/Views/Controls/SampleChooserControl.xaml.cs

```diff
@@ -48,34 +48,40 @@ private void OnLoaded(object sender, RoutedEventArgs e)
 
 		private void OnXamlRootChanged(XamlRoot sender, XamlRootChangedEventArgs args)
 		{
-			if (XamlRoot.Bounds != _initialBounds)
-			{
-				XamlRoot.Changed -= OnXamlRootChanged;
-			}
-			else
-			{
-				Assert.Fail("The XamlRoot's bounds should not have changed after Loaded.");
-			}
+			// TODO MZ: Adjust
+			//if (XamlRoot.Bounds != _initialBounds)
+			//{
+			//	XamlRoot.Changed -= OnXamlRootChanged;
+			//}
+			//else
+			//{
+			//	Assert.Fail("The XamlRoot's bounds should not have changed after Loaded.");
+			//}
```

Comment:
Needs to be done in this PR?

Reply:
I'm not sure what exactly need this fix, but I remember it was blowing up the X11 target, so probably yes.

Reply:
I need to validate if that condition makes sense against WinUI, it is very odd

Reply:
Just curious, about those TODOs, should they be linked to GitHub issues so they are not forgotten?

Reply:
Validated and these assertions are actually not valid and are not in line with WinUI behavior - hence removing